### DISCUSSION TITLE
Automatically select search/ url bar when opening Zen

### DISCUSSION THREAD

**Discussion by mr-cheffy** [zen-browser/desktop#890] [Ideas]
When opening Firefox the search/ url bar is automatically selected means you can just open Firefox and start searching something or openeing a web page, would greatly appreciate if Zen could also adapt that
 
 ~ _Created automatically from issue: https://github.com/zen-browser/desktop/issues/120_

**Comment by atomicptr** [comment]
For me using a very keyboard heavy workflow (eg open browser via keyboard shortcuts) this issue is very annoying. Ironically it works as expected when you create a new tab using Ctrl + T, I just want this to be the same on new windows

**Comment by bancalero** [comment]
For me, the auto select url bar on startup works fine with Multiple Toolbars and Collapsed Toolbar modes, but does not work when using Single Toolbar mode, and as said before, works as expected when creating a new tab. I have started using Single Toolbar today (I like the aesthetics) and this is kinda annoying, as auto selecting the url bar is the default for all the browsers these days and my muscle memory is set in stone to start my browser and type what I need. As someone that does not have the browser open at all times, I found myself typing into nothingness multiple times just in the past 5 hours.
I believe the solution is to just copy the behavior of opening a new tab, right?

**Comment by Asecave** [comment]
I used the workaround from @bancalero and just enabled the top bar, but this no longer works. I tried every combination of hiding the different panels but none of them worked. It also switches to the first tab it can find in any workspace ~3 seconds after opening the browser. Now I have to not only hit a shortcut to start searching, I also have to switch the workspace. I think this issue started appearing after I updated to 1.8.1b.

**Comment by ChryptoCrystal** [comment]
Given the fact this issue only affects those that changed the default settings, by setting zen.urlbar.replace-newtab to False for example, this issue doesn't seem to get enough attention.
I myself depend on having a new tab page on startup and when opening a new tab, as I often utilize the shortcuts I set up on that page, hence I can't leave zen.urlbar.replace-newtab on True. For the times where I'm not starting my browsing through one of those shortcuts, I have to manually click on the search bar, given its not highlighted right away, which is tedious and not like other browsers. Opening a new tab through CTRL+T afterwards WILL have it highlighted, so you can start typing right away, so copying the behavior of that for the very first new tab page on startup would satisfy everyone experiencing this annoyance, so I'm hoping this could eventually be implemented as a solution.
Otherwise I'm absolutely loving this browser and the dedication put into it!

**Reply by ChryptoCrystal** [reply]
This no longer seems to be an issue as of 1.12.5b! So thanks a ton to the developer ❤️ This still happens on the New Tab page when the Browser is manually force stopped and then restarted with previous tabs reappearing automatically, but that's not a real issue.
